flink-formats/flink-sql-protobuf/pom.xml

```diff
@@ -65,6 +65,12 @@ under the License.
 									<include>com.google.protobuf:protobuf-java</include>
 								</includes>
 							</artifactSet>
+							<relocations>
+								<relocation>
+									<pattern>com.google.protobuf</pattern>
+									<shadedPattern>org.apache.flink.shaded.com.google.protobuf</shadedPattern>
```

Comment:
relocate to this path to avoid conflict with other modules: `org.apache.flink.formats.protobuf.shaded.com.google.protobuf`